hack/update-codegen.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+#deep copy
+$GOPATH/bin/deepcopy-gen --input-dirs "github.com/amadeusitgroup/workflow-controller/pkg/api/workflow/v1" -p github.com/amadeusitgroup/workflow-controller/pkg/api/ --bounding-dirs github.com/amadeusitgroup/workflow-controller/pkg/api -h $GOPATH/src/github.com/amadeusitgroup/workflow-controller/hack/boilerplate.go.txt
```

Comment:
This would be more legible if split over several lines.  Just a nit.

Reply:
ok